scripts/us_epa/parent_company/README.md

```diff
@@ -0,0 +1,69 @@
+
+# EPA Parent Company Information and Ownership of Facilities.
+
+This directory contains scripts to import the EPA parent company information and
+facility ownership data from the following GHG table:
+- [`V_PARENT_COMPANY_INFO`](https://enviro.epa.gov/enviro/ef_metadata_html.ef_metadata_table?p_table_name=V_PARENT_COMPANY_INFO)
+
+The generated parent companies are of type `EpaParentCompany` and they are
+attached to a `CensusZipCodeTabulationArea` and `County` (via
+`locatedIn` relation).
+
+## Resolution
+
+We create a Node for each unique Parent Company contained in the table:
+V_PARENT_COMPANY_INFO. However, since there is no unique "id" for these
```

Comment:
could you describe the id scheme in a little more detail here?

Reply:
Done